### ISSUE TITLE
fman does not launch on Arch Linux

### ISSUE THREAD

**Issue by IgorKrupenja** [fman-community/fman#468]
Steps:
1. Install fman 1.1.6 on Manjaro Gnome 17.1.11 either from AUR or by downloading package from fman website.
2. Try launching fman from app icon or from terminal.

Nothing happens, fman does not launch, no output in terminal. Is there anything else i can check to help debug?

![image](https://user-images.githubusercontent.com/26354715/43795721-e71d2ada-9a8a-11e8-8214-fced4de48d00.png)

**Comment by mherrmann** [comment]
What do you get with `/opt/fman/fman`?

**Comment by IgorKrupenja** [comment]
I get this:

```
└─➤ /opt/fman/fman                                       
Could not find platform independent libraries <prefix>
Could not find platform dependent libraries <exec_prefix>
Consider setting $PYTHONHOME to <prefix>[:<exec_prefix>]
Fatal Python error: initfsencoding: unable to load the file system codec
zipimport.ZipImportError: can't find module 'encodings'

Current thread 0x00007f129d50f740 (most recent call first):
[1]    21641 abort (core dumped)  /opt/fman/fman
```

**Comment by mherrmann** [comment]
I'm having the same issue on an up-to-date Arch VM with KDE. It seems to be caused by [PyInstaller not yet supporting Python 3.7](https://github.com/pyinstaller/pyinstaller/issues/3219).

**Comment by zorzysty** [comment]
I'm having similar problem on Antergos KDE:

```
└─➤/opt/fman/fman 
Fatal Python error: initfsencoding: unable to load the file system codec
zipimport.ZipImportError: can't find module 'encodings'

Current thread 0x00007f8bf39d6740 (most recent call first):
[1]    17471 abort (core dumped)  /opt/fman/fman
```

**Comment by IgorKrupenja** [comment]
Please note that I have Python 3.6.6 on my system though.

**Comment by mherrmann** [comment]
It's a problem with the Python version with which fman is built on Arch. I'm working on a fix...

**Comment by mherrmann** [comment]
Fixed in [fman 1.1.7](https://fman.io/changelog#v1.1.7).
